pkg/cmd/cli/cmd/edit.go

```diff
@@ -74,23 +94,25 @@ func NewCmdEdit(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Com
 		},
 	}
 	usage := "Filename, directory, or URL to file to use to edit the resource"
-	kubectl.AddBoundJsonFilenameFlag(cmd, &filenames, usage)
+	kubectl.AddBoundJsonFilenameFlag(cmd, &options.filenames, usage)
 	cmd.Flags().StringP("output", "o", "yaml", "Output format. One of: yaml|json.")
 	cmd.Flags().String("output-version", "", "Output the formatted object with the given version (default api-version).")
+
 	return cmd
 }
 
-// RunEdit contains all the necessary functionality for the OpenShift cli edit command
-func RunEdit(fullName string, f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []string, filenames util.StringList) error {
-	var printer kubectl.ResourcePrinter
-	var ext string
+// Method Complete() completes struct variables
```

Comment:
In Go, the documentation of methods normally starts with the name of the method, without "()":

```go
// Complete completes the struct variables with values that may not be directly provided.
func (o *EditOptions) Complete(...) {...}
```